dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/weld/injection/EEValidationUtils.java

```diff
@@ -71,14 +71,14 @@ private static boolean isProducer(Annotated annotated) {
         return isProducer;
     }
 
-    private static void throwDefinitionException(Class<?> declaringClass, String memberName, Class<?> fieldType) throws DefinitionException {
+    private static void throwDefinitionException(Class<?> declaringClass, String memberName, Class<?> fieldType, String injectedObjectClass) throws DefinitionException {
```

Comment:
Might be better to make the argument a Class rather than a String ... then call Class.getCanonicalName here.